.github/CONTRIBUTING.md

```diff
@@ -20,16 +20,72 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 
 We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/BLACC-Smith/website/issues/new/choose); it's that easy!
 
-## Write bug reports with detail, background, and sample code
+## Create a bug report 
 
-**Great Bug Reports** tend to have:
+Select **Create Issue** from the repository
 
-- A quick summary and/or background
-- Steps to reproduce
-  - Be specific!
-  - Give sample code if you can
-- What you expected would happen
-- What actually happens
-- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+ [![Select button for issue](images/new-issue.png)](https://github.com/blaccsmith/website/issues)
 
-People _love_ thorough bug reports. I'm not even kidding.
+ Select **Get Started** for Bug Report.
+
+  [![Select button for bug report](images/bug-and-feature-request.png)](https://github.com/blaccsmith/website/issues)
```

Comment:
Hey nice updates, however this button is trying to render as a picture. Could you edit this

Reply:
I embedded a link to the image. I can remove it.

Reply:
this has been resolved